submissions/olehmaksymych/html_popup/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"/>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Popup</title>
+    <link rel="stylesheet" href="styl.css" />
+    <link rel="stylesheet" href="styll.css" />
+  </head>
+  <body>
+    <header class="header">
+     
+      <div class="header_checkbox">
+        <div class="header_item">
+          <input type="checkbox" id="popup_box" class="icon_for_thecheckbox" />
+          <label for="popup_box" class="popup_label">
+            <img src="icons/popup-icons/popup-button.png" alt="popup-icon" id="popup_icon"/>
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)